src/psyclone/tests/psyir/backend/visitor_test.py

```diff
@@ -193,7 +193,9 @@ def lower_to_language_level(self):
             # a scope. This is intentional to cause an error during the
             # lowering step.
             self.scope.symbol_table.add(DataSymbol("val", REAL_TYPE))
-            self.replace_with(Return())
+            new_node = Return()
```

Comment:
Please could you update the docstring of this method appropriately and check the copyright date.

Reply:
Done